Recieving of errors when compiling project

#### Expected Behavior:
Project Compiled

#### Actual Behavior:
recieved a list of errors after the prompt: 
Creating cold package with libc libm libpros okapilib, [ERRORS]
arm-none-eabi-g++: error: unrecognized command line option '--whole-archive'; did you mean '--whole-file'?
arm-none-eabi-g++: error: unrecognized command line option '--no-whole-archive,'; did you mean '--no-whole-file'?
arm-none-eabi-g++: error: unrecognized command line option '--gc-sections'; did you mean '--data-sections'?
arm-none-eabi-g++: error: unrecognized command line option '--start-group'
arm-none-eabi-g++: error: unrecognized command line option '--end-group'


#### Steps to reproduce:
Compile with make.

#### System information:
Void Linux

3.1.4

#### Additional Information
I've made this work on void linux in the past, I never recieved this issue

#### Screenshots/Output Dumps/Stack Traces

When running with verbose, I recieve this: 
```arm-none-eabi-g++ -mcpu=cortex-a9 -mfpu=neon-fp16 -mfloat-abi=softfp -Os -g -Wno-psabi -nostdlib -ffunction-sections -fdata-sections -fdiagnostics-color -funwind-tables -Wl,--gc-keep-exported --whole-archive firmware/libpros.a firmware/okapilib.a -lstdc++ --no-whole-archive, -Wl,-T./firmware/v5.ld --gc-sections --start-group ./firmware/libc.a ./firmware/libm.a ./firmware/libpros.a ./firmware/okapilib.a -lc -lm -lgcc -lstdc++ -lsupc++ --end-group -o bin/cold.package.elf,
Creating cold package with libc libm libpros okapilib, [ERRORS]
arm-none-eabi-g++: error: unrecognized command line option '--whole-archive'; did you mean '--whole-file'?
arm-none-eabi-g++: error: unrecognized command line option '--no-whole-archive,'; did you mean '--no-whole-file'?
arm-none-eabi-g++: error: unrecognized command line option '--gc-sections'; did you mean '--data-sections'?
arm-none-eabi-g++: error: unrecognized command line option '--start-group'
arm-none-eabi-g++: error: unrecognized command line option '--end-group'
make: *** [common.mk:230: bin/cold.package.elf] Error 1
```

I haven't modified the common.mk at all

--- a/common.mk
+++ b/common.mk
@@ -7,8 +7,7 @@ GCCFLAGS=-ffunction-sections -fdata-sections -fdiagnostics-color -funwind-tables
 
 WARNFLAGS+=-Wno-psabi
 
-SPACE :=
-SPACE +=
+SPACE := $() $()
 COMMA := ,
 
 DEPDIR := .d

